bin/kano-ui-autostart

```diff
@@ -75,6 +75,9 @@ systemctl --user start kano-home-button
 initflow=`sudo kano-init status`
 if [ "$initflow" != "disabled" ]; then
 
+    # Start the boards daemon so we can talk to hardware periherals during Overture
+    systemctl --user start kano-boards
```

Comment:
Thanks @radujipa ! that is precisely what we need. No animation in place so that the kids do it themselves through the Overture terminal.